lib/rubocop/remote_config.rb

```diff
@@ -55,6 +55,7 @@ def request(uri = @uri, limit = 10, &block)
     def generate_request(uri)
       request = Net::HTTP::Get.new(uri.request_uri)
 
+      request['Authorization'] = "token #{uri.user}" if uri.user
```

Comment:
I'm not sure this really should be Github specific. A Personal Access Token (PAT) can be used in Github using basic auth - which is the syntax suggested here where the user field of the URL is the PAT. Should this line be removed and set the user on the `request` object like this?

```
request.basic_auth(uri.user, uri.password) if uri.user
```

I tested some sample code and this works for Github using a PAT and should work for any URL that supports basic auth.

Reply:
O cool. That works. Didn't know PAT works with basic auth. Thanks